.gitlab-ci.yml

```diff
@@ -44,6 +44,8 @@ variables:
     - schedules
     - web
     - /^[0-9]+$/                   # PRs
+  dependencies:                    []
```

Comment:
what does this do? Please ELI5

Reply:
The default is every job downloads artifacts.
Empty array means it's not needed.
I put this in the template to skip artifacts upload by default.
And it's overriden in two jobs at the bottom.